expression/builtin_math_vec_test.go

```diff
@@ -28,6 +28,9 @@ var vecBuiltinMathCases = map[string][]vecExprBenchCase{
 	ast.Sqrt: {
 		{types.ETReal, []types.EvalType{types.ETReal}, nil},
 	},
+	ast.Abs: {
+		{types.ETDecimal, []types.EvalType{types.ETDecimal}, nil},
```

Comment:
Please add `-benchmem` argument when you run the benchmark so that we can get the memory info like`0 B/op          0 allocs/op`.

Reply:
after add `-benchmem`, I got this
```
BenchmarkVectorizedBuiltinMathFunc/builtinAbsDecSig-VecBuiltinFunc-8        	  500000	      2705 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinMathFunc/builtinAbsDecSig-NonVecBuiltinFunc-8     	   50000	     32222 ns/op	   39936 B/op	     832 allocs/op
```

Reply:
nice.